### ISSUE TITLE
Test fails after 2021-11-27

### ISSUE THREAD

**Issue by bmwiedemann** [requests/toolbelt#270]
Error is maybe from `tests/certs/test_cert.p12`

```
E           ValueError: Client certificate expired: Not After: 2021-11-27 15:28:58Z Not Before: 2018-11-28 15:28:58Z

requests_toolbelt/adapters/x509.py:139: ValueError
________________________ TestX509Adapter.test_x509_pem _________________________
```

Background:
As part of my work on reproducible builds for openSUSE, I check that software still gives identical build results in the future.
The usual offset is +15 years, because that is how long I expect some software will be used in some places.
This showed up failing tests in our `python-requests-toolbelt` package build.
See https://reproducible-builds.org/ for why this matters.

**Comment by bmwiedemann** [comment]
Would be good to see this solved in the next 10 months.

**Comment by pquentin** [comment]
This was fixed in #335 which was merged as part of #336. Thanks for the warning, sorry that we did not act on it in time.
